database/migrations/20220714194550-create-members.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('Members', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      name: {
+        type: Sequelize.STRING
```

Comment:
not null

Reply:
Lo había puesto desde el model, se ve que no lo migró u.u